packages/store/tests/select.spec.ts

```diff
@@ -325,24 +329,78 @@ describe('Select', () => {
       }
     }
 
+    @Injectable()
+    class CustomSelectionStrategy extends SelectionGlobalStrategy {
```

Comment:
should it extend? maybe `implements SelectionStrategy`? as you don't use `super` keyword here

Reply:
the problem is that then you need to implement all the methods in the service